run/libreoffice2john.py

```diff
@@ -46,8 +46,27 @@ def process_file(filename):
         elements = list(r.getiterator())
 
     is_encrypted = False
+    is_supported = True
     key_size = 16
     start_key_generation_name = "SHA1"
+
+    for i in range(0, len(elements)):
+        element = elements[i]
```

Comment:
Minor stylistic comment: `for` loops in Python are for-each loops. So a bit simpler and more idiomatic without the need for indices would be the following.

```suggestion
    for element in elements:
```

Reply:
Thank you for helping review this PR @exploide!